### PR TITLE
Add support for 10.13 High Sierra Apple updates

### PR DESCRIPTION
**Pull request by frlen**

### CODE REVIEW

File: src/simian/mac/munki/plist.py [hunk 1]
```diff
@@ -1563,6 +1563,8 @@ def GetXmlStr(value, indent_num=None):
     str_xml.append('%s<string>%s</string>' % (indent, EscapeString(value)))
   elif value_type is int:
     str_xml.append('%s<integer>%d</integer>' % (indent, value))
+  elif value_type is long:
+    str_xml.append('%s<integer>%d</integer>' % (indent, value))
```
Comment (maximermilov) [inline src/simian/mac/munki/plist.py]: seems unrelated.

Comment (frlen) [reply src/simian/mac/munki/plist.py]: I get this error without it during catalog generation:
File "/base/data/home/apps/.../.../simian/mac/munki/plist.py", line 1590, in GetXmlStr 
raise PlistError('Value type %s not supported: %s', value_type, value) 
PlistError: ('Value type %s not supported: %s', <type 'long'>, 4668317536L)
